common/flatpak-dir.c

```diff
@@ -11422,11 +11425,7 @@ flatpak_dir_list_all_remote_refs (FlatpakDir         *self,
 
   exts = var_summary_get_metadata (summary);
 
-  collection_id = NULL;
-  if (var_metadata_lookup (exts, "ostree.summary.collection-id", NULL, &v) &&
-      var_variant_is_type (v, G_VARIANT_TYPE_STRING))
-    collection_id = var_variant_get_string (v);
-
+  collection_id = var_metadata_lookup_string (exts, "ostree.summary.collection-id", NULL);
```

Comment:
This change doesn't really belong in this commit right?

Reply:
Yeah, it just snuck in due to new generated APIs being available